worlds/timespinner/Locations.py

```diff
@@ -203,7 +202,7 @@ def get_location_datas(world: Optional[MultiWorld], player: Optional[int],
     ]
 
     # 1337156 - 1337170 Downloads
-    if not world or is_option_enabled(world, player, "DownloadableItems"):
+    if not options or options.downloadable_items:
```

Comment:
In what case would `not options` happen?

Reply:
When `get_location_datas` is called by `__inti__.py` on line 47, 
`    location_name_to_id = {location.name: location.code for location in get_location_datas(None, None, None)}`